pyresample/test/test_utils.py

```diff
@@ -11,6 +12,16 @@ def tmp(f):
     return f
 
 
+def tmptiff(width=100, height=100, transform=None, crs=None, dtype=np.uint8):
+    import rasterio
+    array = np.ones((width, height)).astype(dtype)
+    fname = '/vsimem/%s' % uuid.uuid4()
```

Comment:
I understand that you want to create a unique, temporary filename, right ? In this case, I would suggest using the build tempfile library:
```suggestion
    fp, fname = mkstemp()
    del fp
```
And by the way `/vsimem` doesn't exist on my linux system...